src/internal/util/executeSchedule.ts

```diff
@@ -0,0 +1,44 @@
+import { Subscription } from '../Subscription';
+import { SchedulerAction, SchedulerLike } from '../types';
+
+export function executeSchedule(
```

Comment:
I've added two type overloads for this internal function, to make sure we don't use this in a way that would leak with poorly written user-implemented schedulers.